include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -22,7 +22,25 @@ namespace power_grid_model {
 // hide implementation in inside namespace
 namespace math_model_impl {
 
-// block class for the unknown vector and/or right hand side in state estimation equation
+constexpr Idx DISCONNECTED = -1;
```

Comment:
maybe define these values inside the `struct BusInjection` as `static constexpr` and make them small letters?

Reply:
Okay, you also use it for other places besides bus injection. Then just put here. I still don't think we need all capital letters. It is scoped inside namespace.